### ISSUE TITLE
KendoDropdownlist is not setting FormControl value on tab when valuePrimitive is used

### ISSUE THREAD

**Issue by jesulink2514** [telerik/kendo-angular#1382]
## I'm submitting a...

* Bug report

## Current behavior
  * **The Kendo Angular DropDownList component** doesn't set the FormControl value on tab when valuePrimitive is true and the item selected is found using the keyboard (ex. typing the first letter). Visually speaking, the dropdownlist shows the selection but the underlying reactive form is not getting updated.

## Expected behavior
 * Reactive form is updated to reflect visual state.

## Minimal reproduction of the problem with instructions
 * Here is a minimal plunker (forked from documentation sample) that reproduce the issue. https://embed.plnkr.co/TfLi8k4Y3GfwaMj4ye4A
* First, open the dropdown with a mouse click.
![image](https://user-images.githubusercontent.com/2278766/37489393-73cdfcb8-2865-11e8-8208-e9ce9faaeb76.png)
* Press 'f' key to highlight 'Female' option.
![image](https://user-images.githubusercontent.com/2278766/37489403-7db50686-2865-11e8-9692-9a874f445e19.png)
* Press 'tab' key to set selection.
![image](https://user-images.githubusercontent.com/2278766/37489408-8336644c-2865-11e8-98b1-32964bdfd19b.png)
* You can see that model is not getting updated.
* But, if you use the mouse click to select a value, the form is updated.
![image](https://user-images.githubusercontent.com/2278766/37489419-8a12cd64-2865-11e8-9229-953c6245ef2e.png)

## What is the motivation or use case for changing the behavior?
Control is not consistent with user input via keyboard and mouse.

## Environment

Package versions:
```
    "systemjs-json-plugin": "https://unpkg.com/systemjs-plugin-json@0.3.0",
    "@telerik": "https://www.telerik.com/kendo-angular-ui/npm/node_modules/@telerik",
    "@progress": "https://www.telerik.com/kendo-angular-ui/npm/node_modules/@progress",
    "cldr-data": "https://www.telerik.com/kendo-angular-ui/npm/node_modules/cldr-data",
    "@angular": "https://unpkg.com/@angular",
    "angular2-in-memory-web-api": "https://unpkg.com/angular2-in-memory-web-api",
    "hammerjs": "https://unpkg.com/hammerjs@2.0.8",
    "pako": "https://unpkg.com/pako@1.0.5",
    "ts": "https://unpkg.com/plugin-typescript@5.3.3/lib/plugin.js",
    "tslib": "https://unpkg.com/tslib@1.7.1",
    "typescript": "https://unpkg.com/typescript@2.4.2/lib/typescript.js",
    "@angular/common/http": "https://unpkg.com/@angular/common@5.2.2/bundles/common-http.umd.js",
    "@angular/platform-browser/animations": "https://unpkg.com/@angular/platform-browser@5.2.2/bundles/platform-browser-animations.umd.js",
    "@angular/animations/browser": "https://unpkg.com/@angular/animations@5.2.2/bundles/animations-browser.umd.js",
    "@angular/common": "https://unpkg.com/@angular/common@5.2.2",
    "@angular/compiler": "https://unpkg.com/@angular/compiler@5.2.2",
    "@angular/forms": "https://unpkg.com/@angular/forms@5.2.2",
    "@angular/core": "https://unpkg.com/@angular/core@5.2.2",
    "@angular/platform-browser": "https://unpkg.com/@angular/platform-browser@5.2.2",
    "@angular/platform-browser-dynamic": "https://unpkg.com/@angular/platform-browser-dynamic@5.2.2",
    "@angular/upgrade": "https://unpkg.com/@angular/upgrade@5.2.2"
```

Browser:
- Chrome (desktop) version 63.0.3239.132
- Firefox version 56.0
- Edge version 41.16299.15.0

System:
- TypeScript version: 2.4.2
- Node version: v8.7.0 **npm 5.5.1**
- Platform:  Windows

**Comment by valchev** [comment]
@jesulink2514 thank you for bringing this issue to our attention. I will investigate the case further and provide a fix as soon as possible.

**Comment by ipeshev** [comment]
Fixed in v3.0.2-dev.201806051505

**Comment by tsvetomir** [comment]
Fix released in v3.0.2 of the `@progress/kendo-angular-dropdowns`
